src/components/utils/ControlledAlert/ControlledAlert.tsx

```diff
@@ -104,7 +114,13 @@ const ControlledAlert = React.forwardRef<HTMLDivElement, ControlledAlertProps>(
         </Flex>
         {discardable && (
           <Box my={-2} mr={-2} ml={2}>
-            <IconButton aria-label="Discard" variant="unstyled" icon="close" onClick={onClose} />
+            <IconButton
+              aria-label="Discard"
+              variant="unstyled"
+              variantIconColor={variantBackgroundStyle === 'solid' ? 'black' : undefined}
```

Comment:
nit: that should be derived from `useAlertStyles`. All logic related to styling should belong to this hook so we have a centralized place to change stuff 😉

Reply:
FIxed it!